modules/auxiliary/scanner/http/rdp_web_login.py

```diff
@@ -108,30 +108,30 @@ def check_login(rhost, rport, targeturi, domain, username, password, timeout, us
     The timeout is used to specify the amount of milliseconds where a
     response should consider the username invalid."""
 
-    url = f'https://{rhost}:{rport}/{targeturi}'
-    body = f'DomainUserName={domain}%5C{username}&UserPass={password}'
+    url = 'https://' + rhost + ':' + rport + '/' + targeturi
+    body = 'DomainUserName=' + 'domain' + '%5C' + username + '&UserPass=' + password
```

Comment:
This should be domain instead of 'domain' so that it uses the variable not the string 'domain'